### ISSUE TITLE
Convert the quick search field into textual search which gets triggered with the search button

### ISSUE THREAD

**Issue by acorbi** [OpenDevelopmentMekong/wp-odm_tabular_pages#86]
Currently, the **quick filter** functionality allows to filter the contents of the table quickly, without needing to click the **search filter** button. This is currently confusing for the user since it takes 2 different actions to filter the results list.

![screenshot from 2016-12-29 11-55-46](https://cloud.githubusercontent.com/assets/384894/21542501/767d326a-cdbe-11e6-9835-42d53c03483a.png)

For solving this, the **quick search** field can be renamed to **textual search** and the box unified so all filters are triggered only when the user clicks on the search button.

**Comment by prustar** [comment]
@acorbi -- Now it doesn't bring up any results for the example I tried. 

![image](https://cloud.githubusercontent.com/assets/14996905/21557613/17e8152e-ce83-11e6-857e-ed6a1549df0b.png)

I typed it in slow. I hit enter. I typed it then selected the search filter button. No result any which way I do it. As a user this is frustrating when I know the file is there and am likely to google it or search through other means. Which means I've left the site...
